packages/radix-vue/src/Combobox/ComboboxInput.vue

```diff
@@ -63,16 +64,17 @@ function handleInput(event: Event) {
 
   rootContext.isUserInputted.value = true
 }
+
+const pickedProps = reactiveOmit(props, 'autoFocus')
```

Comment:
@zernonia, we would need, I think, also to omit the `as` attribute and pass it manually otherwise, we get this warning

> [Vue warn]: Vue received a Component that was made a reactive object. This can lead to unnecessary performance overhead and should be avoided by marking the component with `markRaw` or using `shallowRef` instead of `ref`.

```vue
const pickedProps = reactiveOmit(props, 'as', 'autoFocus')

// ...

<Primitive
  v-bind="pickedProps"
  :as="props.as"  
```

Reply:
Make sense @iamandrewluca ! Thanks for that! Forgot to consider that `as` could be a component 😅

Instead of converting to reactive object we can forward `as` as is in the composable. 😁

Reply:
The current workaround to get rid of the warning is to use `markRaw`

<img width="381" alt="image" src="https://github.com/radix-vue/radix-vue/assets/1881266/c93b242b-63fd-43dd-98c3-10506be1c78b">

Reply:
I agree with you. Thanks!